src/KalturaUploader.tsx

```diff
@@ -114,37 +60,19 @@ const KalturaUploaderInternal = ({
 
   const client = createClient(endpoint, ks, partnerId);
 
-  const mediaTypeSelectorId = "media-type-selector";
-  const mediaTypeSelector = (
-    <div id={mediaTypeSelectorId}>
-      <p>
-        <strong>Media Type:</strong>
-      </p>
-      {mediaTypeSelectorOptions.map((option) => (
-        <div key={option.id}>
-          <input
-            type="radio"
-            id={`${mediaTypeSelectorId}_${option.id}`}
-            name="mediaType"
-            value={option.id}
-            checked={mediaType === option.value}
-            onChange={(event) => {
-              if (event.target.checked) {
-                setMediaType(option.value);
-              }
-            }}
-          />
-          <label htmlFor={option.id}>{option.label}</label>
-        </div>
-      ))}
-    </div>
-  );
-
   return (
     <div id={kalturaUploaderId}>
       {state.id === "start" && (
         <>
-          {mediaTypeSelector}
+          <MediaTypeSelector
+            idPrefix={kalturaUploaderId}
+            onChange={setMediaType}
+            options={[
+              { id: "audio", label: "Audio", value: MediaTypes.audio },
```

Comment:
Just a very minor thing - do you want to put this array in the module so that it can be reused in Storybook?

Reply:
I did ponder this, however this is really about the configuration of the view, so wanted to keep it strictly in the view layer.